_docs_operate/modules.md

```diff
@@ -87,6 +87,12 @@ For this, the Connector supports the configuration of webhooks which are called
 
 Keep in mind that you need to synchronize the state of the Connector with the Backbone in order to receive events. The `sync` Module automates this, but you can also do this manually by calling the `/api/v2/Account/Sync` route.
 
+### sse (sever sent events) <a href="{% link _docs_operate/configuration.md %}#sse"><i class="fas fa-fw fa-cog"/></a> {#sse}
+
+The sse Module allows the Connector to receive events from the Backbone without exposing the Connector to the Internet.
+This allows the Backbone to communicate required syncs to the connector.
+The sync module is therefore no longer required and can be disabled.
```

Comment:
we should make this required (not allow having sync and sse enabled together)

Reply:
Already dopcumented i will open a PR for the connector